Index: xtables-addons/extensions/GNUmakefile.in
===================================================================
--- xtables-addons.orig/extensions/GNUmakefile.in	2009-06-07 22:10:55.000000000 +0200
+++ xtables-addons/extensions/GNUmakefile.in	2009-06-07 22:11:02.000000000 +0200
@@ -17,7 +17,6 @@
 libdir          := @libdir@
 libexecdir      := @libexecdir@
 xtlibdir        := @xtlibdir@
-kbuilddir       := @kbuilddir@
 
 CC              := @CC@
 CCLD            := ${CC}
@@ -62,7 +61,7 @@
 
 .PHONY: all install clean distclean FORCE
 
-all: subdirs modules user matches.man targets.man
+all: subdirs user matches.man targets.man
 
 subdirs:
 	@for i in ${subdirs_list}; do ${MAKE} -C $$i; done;
@@ -72,7 +71,7 @@
 
 user: ${targets}
 
-install: modules_install subdirs-install ${targets_install}
+install: subdirs-install ${targets_install}
 	@mkdir -p "${DESTDIR}${xtlibdir}";
 	install -pm0755 ${targets_install} "${DESTDIR}${xtlibdir}/";
 
