Last-Update: 2013-06-01
Forwarded: no
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=690613
Author: Pierre Chifflier <pollux@debian.org>
Reviewed-by: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fixh bashism in xt_geoip_dl script

--- a/geoip/xt_geoip_dl
+++ b/geoip/xt_geoip_dl
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-rm -f GeoIPv6.csv{,.gz} GeoIPCountryCSV.zip GeoIPCountryWhois.csv;
+rm -f GeoIPv6.csv GeoIPv6.csv.gz GeoIPCountryCSV.zip GeoIPCountryWhois.csv;
 wget \
 	http://geolite.maxmind.com/download/geoip/database/GeoIPv6.csv.gz \
 	http://geolite.maxmind.com/download/geoip/database/GeoIPCountryCSV.zip;
 gzip -d GeoIPv6.csv.gz;
