From: Helmut Grohne <helmut@subdivi.de>
Subject: consider $ac_tool_prefix for pkg-config

Index: xtables-addons-2.12/configure.ac
===================================================================
--- xtables-addons-2.12.orig/configure.ac
+++ xtables-addons-2.12/configure.ac
@@ -27,7 +27,7 @@
 AC_CHECK_HEADERS([linux/netfilter/x_tables.h], [],
 	[AC_MSG_ERROR([You need to have linux/netfilter/x_tables.h, see INSTALL file for details])])
 PKG_CHECK_MODULES([libxtables], [xtables >= 1.6.0])
-xtlibdir="$(pkg-config --variable=xtlibdir xtables)"
+xtlibdir="$($PKG_CONFIG --variable=xtlibdir xtables)"
 
 AC_ARG_WITH([xtlibdir],
 	AS_HELP_STRING([--with-xtlibdir=PATH],
