Last-Update: 2013-06-01
Forwarded: yes
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=710379
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
From: Denis Ustimenko <den@uzsci.net>
Description: fix quota2/iptables-save invalid record.

--- a/extensions/libxt_quota2.c
+++ b/extensions/libxt_quota2.c
@@ -97,18 +97,18 @@
 quota_mt2_save(const void *ip, const struct xt_entry_match *match)
 {
 	const struct xt_quota_mtinfo2 *q = (void *)match->data;
 
-	if (q->flags & XT_QUOTA_INVERT)
-		printf(" !");
 	if (q->flags & XT_QUOTA_GROW)
 		printf(" --grow ");
 	if (q->flags & XT_QUOTA_NO_CHANGE)
 		printf(" --no-change ");
 	if (q->flags & XT_QUOTA_PACKET)
 		printf(" --packets ");
 	if (*q->name != '\0')
 		printf(" --name %s ", q->name);
+	if (q->flags & XT_QUOTA_INVERT)
+		printf(" !");
 	printf(" --quota %llu ", (unsigned long long)q->quota);
 }
 
 static void quota_mt2_print(const void *ip, const struct xt_entry_match *match,
